backend/service/personalpurchases.service.js

```diff
@@ -40,7 +40,7 @@ const updatePersonalPurchase = async (id, body) => {
     const annotatedPPR = await getPersonalPurchase(id)
     if (body?.status === 'PURCHASED_AND_RECEIPTS_SUBMITTED') {
         sendEmailPPRPurchasedAndReceiptsSubmittedToCoordinator(annotatedPPR)
-    } else if (body?.status === 'REPORTER_REIMBURSE_CONFIRMATION') {
+    } else if (body?.status === 'REPORTER_PAID') {
         sendEmailPPRReimbursedToReporter(annotatedPPR)
```

Comment:
should we do something w.r.t not checking if emails error? would probably be a separate ticket but we should at least log it